src/emmy/collection.cljc

```diff
@@ -128,9 +122,9 @@
 
 (defn- combine [f m1 m2 l-default]
   (letfn [(merge-entry [m e]
-			      (let [k (key e)
```

Comment:
haha, these were left over from a copy-paste from scheme before I converted, clearly... I'm glad you caught one!